tests/distributions/test_transform.py

```diff
@@ -522,7 +533,8 @@ def transform_params(*inputs):
         self.check_vectortransform_elementwise_logp(model)
 
     @pytest.mark.parametrize(
-        "mu,kappa,size", [(0.0, 1.0, (2,)), (floatX(np.zeros(3)), floatX(np.ones(3)), (4, 3))]
```

Comment:
You are introducing a lot of unrelated changes by adding a comma at the end of a function call that then makes black split across multiple lines

Reply:
yeah! I rever them (and created a mess in between but I think we are ok now)